src/pytest_ansible/module_dispatcher/v212.py

```diff
@@ -23,8 +24,6 @@
 # pylint: disable=ungrouped-imports
 if not has_ansible_v212:
     raise ImportError("Only supported with ansible-2.12 and newer")
-else:
-    from ansible.plugins.loader import module_loader
```

Comment:
Why it was not possible to keep the module_loader import line at 27 but without being inside the if? The proposed change will fail to import on old ansible version instead of raising an ImportError.